docs/static/include/pluginbody.asciidoc

```diff
@@ -1237,6 +1237,17 @@ and commit your changes.
 
 To publish version 0.1.0 of your new logstash gem:
 
+If you are using rvm as your ruby version manger, and rvm list shows jruby 1.7.19 as the current version you will need to
+set the following before running bundle exec rake publish_gem.
+export JRUBY_OPTS="--1.9"
+
+If you do not set --1.9 you will get the following errors.
+rake aborted!
+```
```

Comment:
Asciidoc is a bit different from Markdown.  Instead of using triple-backticks to get PRE-style quoting, you should encase the block in a line of dashes (or equals signs), rather like lines 1252-1257  just below:

> If you do not set --1.9 you will get the following errors.
>```
> ----------------------------------
> rake aborted!
> GemPublisher::CliFacade::Error: ERROR:  While executing gem ... (RuntimeError)
>     Can't find 'rubygems-update' in any repo. Check `gem source list`.
> ----------------------------------
>```

Don't worry about pruning extra lines (outside of the PRE block, anyway).  Asciidoc prefers them, and cleans up appropriately at render time.